platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js

```diff
@@ -101,15 +102,18 @@ export default class RetrieveMetadataLoaderAsync extends RetrieveMetadataLoader
 
   async preLoad() {
     const preLoaders = this.getPreLoaders();
-    const result = await this.runLoaders(preLoaders);
+    const seriesData = await this.runLoaders(preLoaders);
```

Comment:
This is better than "result", but what is "seriesData"?

Reply:
It's the result of the QIDO-RS Search For Series request, but that's too long to write.

Reply:
Added a comment.